onert-micro/onert-micro/include/pal/common/PALArithmeticOpCommon.h

```diff
@@ -65,6 +65,37 @@ OMStatus ArithmeticOp(const core::BinaryArithmeticBroadcastParams &params, const
   return Ok;
 }
 
+template <typename T, typename Fn>
+OMStatus QuantizedArithmeticOp(
+  const core::BinaryArithmeticBroadcastParams &params, const int flat_size,
+  const onert_micro::core::QuantizationParams &input1_qparams, const T *input1_data,
+  const onert_micro::core::QuantizationParams &input2_qparams, const T *input2_data,
```

Comment:
Not sure. But may it would be better to group `q_params with input_data` into `QuantizedData`?